doc/nrf/known_issues.rst

```diff
@@ -1178,6 +1178,12 @@ KRKNWK-6255: RSSI parameter adjustment is not applied
 SoftDevice Controller
 =====================
 
+.. rst-class:: v1-7-0 v1-6-1 v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
+
+DRGN-15903: BT_CTLR_TX_PWR is ignored by the SoftDevice Controller
+
+  **Workaround:** Use the HCI API :c:func:`sdc_hci_cmd_vs_zephyr_write_tx_power()` to dynamically set TX power.
```

Comment:
We could consider making the workaround more generic so that it also applies on nRF53.
```suggestion
  **Workaround:** Use the HCI command Zephyr Write Tx Power Level to dynamically set TX power.
```